src/plugins/kibana/server/routes/api/ingest/register_simulate.js

```diff
@@ -1,24 +1,67 @@
 import _ from 'lodash';
 import handleESError from '../../../lib/handle_es_error';
-import handleResponse from '../../../lib/process_es_ingest_simulate_response';
-import handleError from '../../../lib/process_es_ingest_simulate_error';
-import simulateRequestSchema from '../../../lib/schemas/simulate_request_schema';
-import ingestSimulateApiKibanaToEsConverter from '../../../lib/converters/ingest_simulate_api_kibana_to_es_converter';
+import simulateSchema from '../../../lib/ingest/simulate/schema';
+import simulateConverter from '../../../lib/ingest/simulate/converter';
 import { keysToCamelCaseShallow, keysToSnakeCaseShallow } from '../../../../common/lib/case_conversion';
 
+export function handleResponse(resp) {
```

Comment:
Now that I think about it, couldn't this handleResponse function be considered the `esToKibana` converter for simulate?